packages/core/core-flows/src/product/workflows/update-product-variants.ts

```diff
@@ -151,6 +154,32 @@ export const updateProductVariantsWorkflow = createWorkflow(
 
     const updatedVariants = updateProductVariantsStep(updateWithoutPrices)
 
+    const variantsToDismissInventory = transform(
+      { input, updatedVariants },
+      (data) => {
+        const variantIds: string[] = []
+
+        if ("product_variants" in data.input) {
+          variantIds.push(
+            ...data.input.product_variants
+              .filter((v) => v.id && v.manage_inventory === false)
+              .map((v) => v.id!)
+          )
```

Comment:
nit: loop -> push instead of loop -> filter -> map -> create array -> spread array